utils/sl/fake_scons/SCons/Warnings/__init__.py

```diff
@@ -19,7 +19,13 @@
 # SOFTWARE.
 """Fake scons environment shutting up pylint on SCons files"""
 
+class Warning:
```

Comment:
(pylint-redefined-builtin) Redefining built-in 'Warning'
(pylint-missing-docstring) Missing class docstring